configure.ml

```diff
@@ -1079,6 +1079,31 @@ let write_dbg_wrapper f =
 
 let _ = write_dbg_wrapper "dev/ocamldebug-coq"
 
+(** * Generate an implementation of 63-bit arithmetic *)
+
+let ml_file_copy input output =
+  let i = open_in input in
+  let o = open_out output in
+  let pr s = fprintf o s in
+  pr "(* DO NOT EDIT THIS FILE: automatically generated by ../configure *)\n";
+  pr "(* see kernel/uint63_amd64.ml and kernel/uint63_x86.ml *)\n";
+  try
+    while true do
+      output_string o (input_line i); output_char o '\n'
+    done
+  with End_of_file ->
+    close_in i;
+    close_out o;
+    Unix.chmod output 0o555
+
+let write_uint63 () =
```

Comment:
I'd just use `cp`, what is the point of a pure ML implementation of `cp` ?

Reply:
It's not exactly `cp`, it also adds the header. I could use `cat`, but I'm not sure we are going to get something portable. I'm not against it, though, but would focus on the other parts of the patch first.